packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vm-console-certificates/tasks/main.yml

```diff
@@ -54,14 +54,18 @@
     copy:
       content: "{{ ovirt_ca_key }}"
       dest: "{{ ovirt_vmconsole_ca_file }}"
+      owner: 'ovirt-vmconsole'
+      group: 'ovirt-vmconsole'
       mode: 0644
 
   - name: Add cert files
     copy:
       src: "{{ ovirt_pki_dir }}/certs/{{ ovirt_vds_hostname }}-ssh-cert.pub"
       dest: "{{ ovirt_vmconsole_cert_file }}"
       remote_src: no
-      mode: preserve
+      owner: 'ovirt-vmconsole'
+      group: 'ovirt-vmconsole'
```

Comment:
Why to set it to ovirt-vmconsole owner and group?  It shouldn't be writable by the vmconsole sshd process, read-only access is enough.

The same applies in the change above.  I think all we need in this patch is setting the correct permissions.

Reply:
i think it makes sense to align with ownership on engine, it's the same package/project